src/app/directory/directory.component.html

```diff
@@ -1,24 +1,23 @@
 <header>header</header>
 
 <div class="content" role="main">
```

Comment:
Not sure why it removed then added them again... All i did was do a clean clone and added my parts and the component where needed.